### PR TITLE
Project 5 - Guess Who?

### PR DESCRIPTION
**Pull request by AsaHildebrand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Project Name
 
-Replace this readme with your own information about your project.
+This weeks project was to creat our own version of the board game Guess Who? using HTML, CSS and Javascript.
```
Comment (petercreations) [inline README.md:3]: Well done on this project and achieving the goals! :)

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started to go trough all the material about, for example, object, arrays and loops. Then I started right away to manage to do as much as possible in the beginning of the week to have time left if I was needing help. I solved the first stepd by my self and then I asked my team for help. If I had more time I would have changed the desig more than the background color and also changed the persons and attributes to something else.
```
Comment (petercreations) [inline README.md:8]: Nice descriptive README ! 😀

File: code/index.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
       <img
         class="guess-who-icon"
         src="images/guess-who-bubble.png"
-        alt="Guess Who"
+        alt="Guess Who?"
```
Comment (petercreations) [inline code/index.html:20]: Using "Guess Who Icon" instead could also add more descriptiveness to the alt tag if the img did not display

File: code/index.html [hunk 4]
```diff
@@ -26,18 +26,29 @@ <h1>Does the person have</h1>
         <optgroup label="hair color">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+           <option value="hidden">hidden hair</option>
```
Comment (petercreations) [inline code/index.html:29]: A small thing, this line could match the indentation of the rest of the option tags

File: code/index.html [hunk 5]
```diff
@@ -26,18 +26,29 @@ <h1>Does the person have</h1>
...
+          <option value="grey">grey hair</option>
+          <option value="black">black hair</option>
+          <option value="brown">brown hair</option>
+          <option value="orange">orange hair</option>
+          <option value="purple">purple hair</option>
+          <option value="white">white hair</option>
```
Comment (petercreations) [inline code/index.html:35]: Great job adding more options !

File: code/script.js [hunk 6]
```diff
@@ -251,93 +265,202 @@ const setSecret = () => {
...
+// We also need a variable that stores the actual value of the question we've selected.
+const value = questions.value
+console.log(value)
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
+      value: value,
       // 👆 add the value from the input here
       category: category,
```
Comment (petercreations) [inline code/script.js:290]: Could be simplified from `category:category, `to just   `category,` for any lines matching this criteria

**Review by petercreations — APPROVED** [review]
Great job over all! :) 

Some thoughts, more comments could be removed and simplified for each section for a clearer overview of each section. 

It is also possible to remove all `console.log ` statements that were used during the build of the project for testing for when ready to publish the final site.

File: code/script.js [hunk 7]
```diff
@@ -251,93 +265,202 @@ const setSecret = () => {
...
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
-
+// This variable stores what option group (category) the question belongs to.
+  
+// We also need a variable that stores the actual value of the question we've selected.
+const value = questions.value
+console.log(value)
```
Comment (KarolCislowski) [inline code/script.js:284]: remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

I know it was an extra point of this assignment to style it your way but remember that there is no another way to master CSS then practice it over and over again 🦾 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**